pontoon/teams/views.py

```diff
@@ -210,6 +214,46 @@ def request_projects(request, locale):
     return HttpResponse('ok')
 
 
+@login_required(redirect_field_name='', login_url='/403')
+@require_POST
+def request_locales(request):
+    """Request locales to be added to pontoon."""
+    form = LocaleRequestForm(request.POST)
+    code = request.POST['code']
```

Comment:
As a general advice, you will want to access the validated form data from the `form.cleaned_data` dictionary rather than using _raw_ POST data. This way you ensure the data you are using is consistent, and cleaned according to the field's validation rules. Please note such dictionary is populated once `form.is_valid()` is called. You can read more about [`Form.cleaned_data` in Django docs](https://docs.djangoproject.com/en/1.11/ref/forms/api/#django.forms.Form.cleaned_data).